packages/react-core/src/components/Dropdown/DropdownWithContext.tsx

```diff
@@ -98,28 +144,45 @@ export class DropdownWithContext extends React.Component<DropdownProps & OUIAPro
               {React.Children.map(toggle, oneToggle =>
                 React.cloneElement(oneToggle, {
                   parentRef: this.baseComponentRef,
+                  getMenuRef: this.getMenuComponentRef,
                   isOpen,
                   id,
                   isPlain,
                   'aria-haspopup': ariaHasPopup,
                   onEnter: () => this.onEnter()
                 })
               )}
-              {isOpen && (
-                <DropdownMenu
-                  component={component}
-                  isOpen={isOpen}
-                  position={position}
-                  aria-labelledby={contextId ? `${contextId}-toggle` : id}
-                  openedOnEnter={openedOnEnter}
-                  isGrouped={isGrouped}
-                  autoFocus={openedOnEnter && autoFocus}
-                >
-                  {renderedContent}
-                </DropdownMenu>
-              )}
+              {menuAppendTo === 'inline' && isOpen && menuComponent}
             </BaseComponent>
           );
+          return menuAppendTo === 'inline' ? (
+            mainComponent
+          ) : (
+            <PopoverBase
+              content={popoverContent}
+              onCreate={(tip: TippyInstance) => (this.tip = tip)}
+              isVisible={isOpen}
+              trigger={'manual'}
+              arrow={false}
+              interactive
+              interactiveBorder={0}
+              maxWidth="none"
+              distance={0}
+              appendTo={menuAppendTo}
+              boundary="window"
+              flip={false}
+              placement={this.getPlacement(position, direction)}
+              hideOnClick={false}
+              theme="pf-popover"
```

Comment:
I wonder if we should use `css(styles...)` here?

Reply:
not needed since it is just one class, and i cannot use styles object since tippy decided it needs to add the `-theme` suffix to the passed in theme class here. This class (`pf-popover-theme` lives in the tippy-overrides.css in react-styles